frontend/src/ui/pages/ErrorPage.js

```diff
@@ -23,20 +23,27 @@ class ErrorPage extends React.Component {
     let title = ''
     let message = ''
     if (this.state.error === 'noWeb3Render') {
-      title = 'No MetaMask found'
+      title = 'Wallet not found'
       message = (
         <span>
-          This application requires MetaMask extension for Google Chrome.
+          This application requires a web browser Wallet extension.
           <br />
-          Please make sure you are running the latest version of Google Chrome and follow this link
-          to install MetaMask.
+          Check{' '}
+          <a
+            href="https://poa.network/dapps?category=wallets"
```

Comment:
Replaced the link to the Google Chrome extension with a link to [this page](https://poa.network/dapps?category=wallets) in POA's web.

Since we don't want to specify a browser/extension, that link may be suitable because it links to Nifty Wallet and MetaMask (just in case the user is using Chrome or Firefox).

BTW this is how it looks:

![walletnotfound](https://user-images.githubusercontent.com/4421917/45515897-de1d6c80-b77f-11e8-8027-aa90f1aebf28.png)

Reply:
@phahulin @pablofullana does this meet the expectation?